app/app/settings.py

```diff
@@ -544,3 +544,6 @@
             'name': 'Index View',
         }]
     SILKY_MAX_RECORDED_REQUESTS_CHECK_PERCENT = env.int('SILKY_MAX_RECORDED_REQUESTS_CHECK_PERCENT', default=10)
+
+# Gitcoin addresses
+GITCOIN_TIP_ADDRESS = env('GITCOIN_TIP_ADDRESS', default='0x00de4b13153673bcae2616b67bf822500d325fc3')
```

Comment:
@mbeacom @owocki should we default it to some particular address or would it be better to default it to a blank string? I would prefer blank string to avoid sending it to someone else's address even when I have pulled it locally and not set anything from my end.

Reply:
Any word on this? Looks like the last thing that needs to be decided

Reply:
Please default to a blank string for this.  `GITCOIN_TIP_ADDRESS = env('GITCOIN_TIP_ADDRESS', default='')` and add an entry to `docs/ENVIRONMENT_VARIABLES.md` following the syntax in the MD file.

Reply:
This should default to `''` and update `docs/ENVIRONMENT_VARIABLES.md` to include the additional table values detailing the env var.

Reply:
Done